### ISSUE TITLE
How to use structured data for FASHION TIPS?

### ISSUE THREAD

**Issue by shopstye** [schemaorg/schemaorg#4308]
Hello, i have website for **Fashion tips** presented from us. How to use structured data and wich type for my website for this type of data to be more relative for google?

**Comment by gmackenz** [comment]
I would suggest reading this to start with: 

- https://schema.org/docs/gs.html
- https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data

**Comment by rrlevering** [comment]
Yeah, agreed on those suggestions from Gordon.  It sounds like it's going to mostly be articles(?) so just using generic structured data like Article (you can throw some categories on there) and maybe BreadcrumbList.  These are usually included in most CMS systems by default these days.
